btcstaking-tracker/stakingeventwatcher/stakingeventwatcher.go

```diff
@@ -437,13 +438,24 @@ func (sew *StakingEventWatcher) watchForSpend(spendEvent *notifier.SpendEvent, t
 		// As we only care about unbonding transactions, we do not need to take additional actions.
 		// We start polling babylon for delegation to stop being active, and then delete it from unbondingTracker.
 		sew.logger.Debugf("Spending tx %s for staking tx %s is not unbonding tx. Info: %v", spendingTxHash, delegationId, err)
+		proof, err := sew.waitForStakeSpendInclusionProof(quitCtx, spendingTx)
+		if err != nil {
+			sew.logger.Errorf("unbonding tx %s for staking tx %s proof not built", spendingTxHash, delegationId)
+			return
+		}
+		sew.reportUnbondingToBabylon(quitCtx, delegationId, spendingTx, proof)
 		sew.waitForDelegationToStopBeingActive(quitCtx, delegationId)
```

Comment:
we probably do not need it here any more, as after succesful unbonding report delegation won't be active (and `reportUnbondingToBabylon` has acitvity check inside)